### DISCUSSION TITLE
Having issues getting things to work

### DISCUSSION THREAD

**Discussion by alexwachowski** [community/community#103069] [New to GitHub]
could someone look through my most recent repository and see if I've done something wrong? I'm new and I followed the steps I saw in a tutorial but my code isn't working and I know it works when I run node on my local computer. (https://github.com/alexwachowski/my-blog) Thanks in advance!

**Comment by queenofcorgis** [comment]
Hi @alexwachowski , we're happy you're here! You are more likely to get a useful response like what errors you are receiving or what specifically isn't working, giving a few more details might help someone give you a nudge in the right direction.

**Reply by alexwachowski** [reply]
currently I'm having issues getting anything to work that isn't specifically in index.html or style.css. I've been learning and using a lot of JS (react and recently three.js) but when I upload them here they don't work and can only get the read me or whatever is in the base index.html to appear when running the site. 

here's an example of what I'm seeing: https://alexwachowski.github.io/testing/ 
currently it's just what's in the index.html file but I have a bunch going on in the index.js file for three.js stuff. 
https://github.com/alexwachowski/testing
